src/content/blog/2024/special_interest_groups.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: Special Interest Groups
+subtitle: A new place to collaborate with others working in your field
+pubDate: 2024-03-02T00:00:00+01:00
+#pubDate: 2024-03-12T00:00:00+01:00
+headerImage: https://images.unsplash.com/photo-1483428400520-675ef69a3bc4
+headerImageAlt: 'Photo by Hudson Hintze on Unsplash'
+authors:
+    - 'ewels'
+label:
+    - 'community'
+embedHeaderImage: false
+---
+
+import crossFunctional from '@assets/images/blog/special_interest_groups/cross-functional-organisation.png';
+
+import { Image } from 'astro:assets';
+
+Today, we're very happy to announce a new nf-core initiative:
+_Special Interest Groups_.
+
+# Background
+
+Until now, nf-core has been very focused on doing what we do best:
+creating analysis pipelines and the tooling around them.
+We have structured ourselves around this: Slack channels for every pipeline
+bring together people working on the same set of tools to collaborate.
+It's clear to newcomers where to find help and it keeps the discussion focussed.
```

Comment:
I tried to rephrase it a little from @mashehu comment 
> this whole paragraph reads bumpy
```suggestion
Until now, we have structured the nf-core community around this: Slack channels for every pipeline that
bring together people working on the same set of tools to collaborate.
This, makes clear to newcomers where to find help and also keeps the discussion focussed.
```

Reply:
Thanks! I ended up doing a major carve and deleted most of it in the end.